pr_watch/migrations/0011_auto_20200728_1807.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.12 on 2020-07-28 18:07
```

Comment:
@Kelketek, nit: can you rename the migration file to a meaningful one?

Reply:
Done.